db/schema.rb

```diff
@@ -613,6 +613,7 @@
     t.string "identified_by", null: false
     t.bigint "consultee_id"
     t.boolean "consultation_required", default: true, null: false
+    t.string "status", default: "success", null: false
```

Comment:
I think we need a default status of `pending` so that we can re-run the job and not fetch them again.